### ISSUE TITLE
Integration with CI systems

### ISSUE THREAD

**Issue by robotpilot** [ROBOTIS-GIT/dynamixel-workbench#117]
- https://github.com/ROBOTIS-GIT/dynamixel-workbench/commit/d58e063fc5ae93294373a7209ce08fe0baa9dfa0#commitcomment-27684734
- https://travis-ci.org/ROBOTIS-GIT/dynamixel-workbench/jobs/343626574#L166
- https://github.com/ros/rosdistro/pull/17019
- https://discourse.ros.org/t/new-packages-for-kinetic-2018-02-22/4032

**Comment by robotpilot** [comment]
https://travis-ci.org/ROBOTIS-GIT/dynamixel-workbench

**Comment by ROBOTIS-OpenSourceTeam** [comment]
https://github.com/ROBOTIS-GIT/dynamixel-workbench/issues/112

**Comment by robotpilot** [comment]
ref Issues: https://github.com/ROBOTIS-GIT/dynamixel-workbench/issues/112

**Comment by robotpilot** [comment]
@tfoote
I applied the Travis CI to solve the problem that occurred before. 
https://travis-ci.org/ROBOTIS-GIT/dynamixel-workbench
I really appreciate you. I am ashamed of myself. 
In addition to this package, I will apply Travis CI to all the packages we have developed.
http://repositories.ros.org/status_page/ros_kinetic_default.html?q=pyo

@130s 
I am so glad to have introduced you a nice solution. Thank you very much. I will apply it for our all packages and I will try to contribute to it. Thank you very much.
https://github.com/ros-industrial/industrial_ci
